nautobot/docs/release-notes/version-2.1.md

```diff
@@ -8,6 +8,13 @@ This document describes all new features and changes in Nautobot 2.1.
 
 ### Added
 
+#### isnull Filter on Nullable Fields
```

Comment:
Perhaps?

```suggestion
#### UI/API `isnull` Filter on Nullable Fields
```

in which case this would alphabetically belong at the end of the `Added` section rather than the beginning of it?

Reply:
I think the issue is ordered by their issue number perhaps? This issue is 1905 and should be in front

Reply:
The previous minor versions' *release overviews* (not the summary release-notes for each patch) are ordered alphabetically.